ecommerce-ui/src/ShoppingCart.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './App.css';
+import Rental from './Rental/Rental.js';
+import PropTypes from 'prop-types';
+
+
+class ShoppingCart extends React.Component {
+    static propTypes = {
+        rentals: PropTypes.array.isRequired,
+        onDeleteRental: PropTypes.func.isRequired
+      }
+    
+    render() {
+        let totalCost = 0;
+        const rentalList = this.props.rentals
+            .map((rental, idx) => {
+                totalCost += parseInt(rental.payment.cost);
+                return (
+                        <Rental  className="shopping-rental"
+                            rental={rental} 
+                            key = {idx}
+                            idx = {idx}
+                            onClick={this.props.onDeleteRental}
+                            buttonText="Delete"
+                        />
+
+                );
+            });
+
+        return (
+            (rentalList.length > 0 )
+            ? <div className="shopping-div">
```

Comment:
You can also render an inline if with the && operator, leaving out the : null at the end.